cmd/aws-event-sources-controller/Dockerfile

```diff
@@ -1,20 +1,29 @@
-FROM golang:1.14-alpine AS builder
+FROM golang:1.14-stretch AS builder
 
 ENV CGO_ENABLED 0
 ENV GOOS linux
 ENV GOARCH amd64
 
 WORKDIR /go/src/aws-event-sources-controller
 
-RUN apk add --no-cache openssh-client ca-certificates
+COPY go.mod go.sum ./
+RUN go mod download
 
-COPY ./ .
-RUN go install -installsuffix cgo ./cmd/aws-event-sources-controller
+COPY . .
+RUN BIN_OUTPUT_DIR=/bin make aws-event-sources-controller && \
+    mkdir /kodata && \
+    mv .git/* /kodata/ && \
```

Comment:
Honestly I am not sure what's going on with the `kodata` folder. Should be added for the other repo's as well?

Reply:
The Knative logger falls back to not logging the current SHA/tag if that's missing, so kodata is optional.
I personally like being able to filter logs by code revision in Kibana (that's what that feature is for), so it could make sense to have it by default on the long run.